README.md

```diff
@@ -72,6 +73,8 @@ Assist Adnan by writing a program, using any allowed language, to find the city
 
 ### INPUT
 
+You can find an example of the input file [here](./input.txt). This file contains 1 million line to run your tests and validate your solution.
```

Comment:
@chermehdi What is the expected file location ? same for output file if any

Reply:
Output should be a file named `output.txt` in the working directory of the process